src/main/java/com/hubspot/imap/protocol/message/ImapMessage.java

```diff
@@ -25,9 +29,10 @@
   long getGmailThreadId() throws UnfetchedFieldException;
   Set<GMailLabel> getGMailLabels() throws UnfetchedFieldException;
   Message getBody() throws UnfetchedFieldException;
+  String bodyToString() throws IOException;
 
   class Builder implements ImapMessage {
-    private static DateTimeFormatter INTERNALDATE_FORMATTER = DateTimeFormatter.ofPattern("d-MMM-yyyy HH:mm:ss Z");
+    public static final DateTimeFormatter INTERNALDATE_FORMATTER = DateTimeFormatter.ofPattern("d-MMM-yyyy HH:mm:ss Z");
```

Comment:
if this needs to be used elswhere, then we should probably move it to another class entirely. Importing from inner classes is frowned  upon.